change color scheme

CodebyKhayla opened the pull request:
First attempt at making the blog color scheme more in line with main matplotlib site.

khaylacode committed:
change color scheme

First attempt at making the blog color scheme more in line with main matplotlib site.

davidevaleriani commented:
I think the orange-ish color as background for the menu has a contrast that is too high. Let me try to improve this a bit and then we can merge

Commit:
Another iteration of the blog

- added images to the blog posts
- added tutorials category and sample post
- added custom links for the menu, currently only matplotlib added, but we could add extra ones from config.toml
- changed background color of the menu to green-ish
- added matplotblog logo in each page
- removed button to go back to the homepage from each post, as the matplotblog logo is always active
- removed suggestion for the next post in the page of each post
- changed links to orange-ish

khaylacode committed:
Iteration of design changes

-added orange accent to muted text
-changed name of Matplotlib link to 'About'

khaylacode committed:
A New Blog Intro Post

-First draft of post introducing the blog.

davidevaleriani committed:
Merge pull request #2 from matplotlib/a-new-blog-intro-post

A New Blog Intro Post
